Description: Correct linking order to prevent FTBFS with GCC4.5 + binutils-gold.
Author: Alessio Treglia <quadrispro@ubuntu.com>
Forwarded: Yes
---
 src/makefile |   15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

--- a/src/makefile
+++ b/src/makefile
@@ -95,19 +95,22 @@
 #
 
 ../bin/applyplugin:	applyplugin.o load.o default.o
-	$(CC) $(CFLAGS) $(LIBRARIES) $(LDFLAGS)				\
+	$(CC) $(CFLAGS)							\
 		-o ../bin/applyplugin					\
-		applyplugin.o load.o default.o
+		applyplugin.o load.o default.o				\
+		$(LIBRARIES) $(LDFLAGS)
 
 ../bin/analyseplugin:	analyseplugin.o load.o default.o
-	$(CC) $(CFLAGS) $(LIBRARIES) $(LDFLAGS)				\
+	$(CC) $(CFLAGS) 						\
 		-o ../bin/analyseplugin 				\
-		analyseplugin.o load.o default.o
+		analyseplugin.o load.o default.o			\
+		$(LIBRARIES) $(LDFLAGS)
 
 ../bin/listplugins:	listplugins.o search.o
-	$(CC) $(CFLAGS) $(LIBRARIES) $(LDFLAGS)				\
+	$(CC) $(CFLAGS) 						\
 		-o ../bin/listplugins	 				\
-		listplugins.o search.o
+		listplugins.o search.o					\
+		$(LIBRARIES) $(LDFLAGS)
 
 ###############################################################################
 #
